README.md

```diff
@@ -87,6 +94,54 @@ for the SSE 4.2 implementation.
 For ARM64 support Nightly Rust is needed and the crate feature `aarch64_neon` needs to be enabled. CAVE: If this features is
 not turned on the non-SIMD std library implementation is used.
 
+### WASM32
+For wasm32 support, the crate feature `wasm32_simd128` needs to be enabled.  CAVE: If this feature is not turned on, then
+the non-SIMD std library implementation is used.
+
+#### Building/Targeting WASM
+To build/target WASM outside a `wasm-pack` context, you can do the following:
+
+* Install toolchain with `wasm32-wasi` or `wasm32-unknown-unknown` (e.g. `rustup target add wasm32-wasi`).
+* Install a WASM runtime (e.g. [Wasmer]/[Wasmtime]/[WAVM]).
+* Install `wasm-runner` a simple runner wrapper to run WASM targeted code with a WASM runtime:
+
+```
+$ cargo install wasm-runner
+```
+
+* Add a Cargo configuration file to target `wasm` and allow the unit tests to be run with a WASM VM *by default*:
+
+```
+[build]
+target = "wasm32-wasi"
+rustflags = "-C target-feature=+simd128"
+
+[target.'cfg(target_arch="wasm32")']
+runner = ["wasm-runner", "wasmer"]
+```
+
+* Run the build/tests with the WASM `wasm32_simd128` feature or all features:
+
+```
+$ cargo test --features wasm32_simd128
+$ cargo test --all-features
+```
+
+You can do this without configuration as well:
+
+```
+$ RUSTFLAGS="-C target-feature=+simd128" \
+    CARGO_TARGET_WASM32_WASI_RUNNER="wasm-runner wasmer" \
+    cargo test --target wasm32-wasi --features wasm32_simd128
+$ RUSTFLAGS="-C target-feature=+simd128" \
+    CARGO_TARGET_WASM32_WASI_RUNNER="wasm-runner wasmer" \
+    cargo test --target wasm32-wasi --all-features
```

Comment:
Running the units tests is not really that important for the README as I expect that it most developers just want to use simdutf8 in an existing WASM32 project. Maybe extract that to a wasm32-testing.md file and reference it here?

Reply:
That is a good point--I will move this and refer to it.

Reply:
Moved and cleaned up a bit in 66057f1